### DISCUSSION TITLE
Cannot find module 'node:readline/promises' or its corresponding type declarations [@types/node]

### DISCUSSION THREAD

**Discussion by chialuka** [DefinitelyTyped/DefinitelyTyped#58451] [Issues with a @types package] [answered]
Importing the promised based API of the readline module like this:
`import * as readline from "node:readline/promises"`
 results in this error from the TypeScript compiler:
`Cannot find module 'node:readline/promises' or its corresponding type declarations.ts(2307)`

Only works when I import the callback and sync API: `import * as readline from "node:readline`

[a comment by an automated account is not shown]

**Comment by nickserv** [comment, marked as answer]
Looks like this has been fixed.
